lib/daylight/associations.rb

```diff
@@ -92,9 +92,10 @@ def has_many name, options={}
     # ActiveResource::Associations#belongs_to
 
     def belongs_to name, options={}
+
       create_reflection(:belongs_to, name, options).tap do |reflection|
 
-        nested_attribute_key = "#{reflection.name}_attributes"
+        nested_attribute_key = "#{reflection.name}"
```

Comment:
explain?

Reply:
Created reflections don't have the "_attributes" suffix anymore. We were just sending a "merchant" over "merchant_attributes" so we failed everytime.